plugins/woocommerce/client/admin/client/settings-payments/components/icons/badge-official.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { SVG, Path } from '@wordpress/primitives';
+
+export default (
```

Comment:
Why did you choose `tsx` over compressed `svg`?

Reply:
I wanted to keep things self-contained and avoid the initial image load. And it also seems this is the pattern we are using in our components. But I agree that it can go both ways.

Reply:
During the NOX project I was told by some folks in Ghidorah that the best practice now is to directly reference the SVG, rather than importing it as a component, for example: 

https://github.com/woocommerce/woocommerce/blob/3a19457bb65eaf7c53e4a7b7267b9a8d890af6b5/plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx#L131-L140

I found this related P2 that explains the rationale: pdaBTz-lC-p2

But agree that this pattern is also used heavily in our components.

Reply:
Ok. I am sold. [Changed](https://github.com/woocommerce/woocommerce/pull/54955/commits/3b2966158252ca5cefb2c91e9941d726600186e8) it to using a SVG file.